lib/translator/locale-parser.js

```diff
@@ -37,10 +37,17 @@ function getPreferredLocale(acceptLanguage, languages, defaultLocale) {
 function getLocales(acceptLanguage) {
     const localeObjects = AcceptLanguageParser.parse(acceptLanguage);
 
-    return _.map(localeObjects, localeObject => {
+    const localesArr = _.map(localeObjects, localeObject => {
```

Comment:
How about just `locales` instead of `localesArr`?

Reply:
fixed.  I guess the Arr is unnecessary.